homeassistant/components/velbus/config_flow.py

```diff
@@ -74,15 +125,11 @@ async def async_step_usb(self, discovery_info: UsbServiceInfo) -> ConfigFlowResu
         await self.async_set_unique_id(
             f"{discovery_info.vid}:{discovery_info.pid}_{discovery_info.serial_number}_{discovery_info.manufacturer}_{discovery_info.description}"
         )
```

Comment:
One thing is that we want unique_ids to be unique, but if I understand correctly, we can't discover USB devices on the network. So I am not 100% sure what the best approach is. Maybe the best idea is to just migrate it to use serial number as both have them (if I saw correctly)

Reply:
or leave the unique id out of it, after all we already check if we use the same connection string

Reply:
But if we plug in the same device in a different usb port?

Reply:
not sure what the best approach is ...

Reply:
probably yes, will change it to use the serial

Reply:
This also requires a migration by the way